docs/guide/usage.md

```diff
@@ -157,3 +157,7 @@ Next, ensure the `tsconfig.json` of your host application is set up to properly
   ]
 }
 ```
+
+## Icons
+
+Kongponents sources all icons from [@kong/icons](https://github.com/Kong/icons) library. If you wish to use standalone icons in your project please refer to the documentation for instructions.
```

Comment:
```suggestion
Kongponents sources its icons from the [@kong/icons](https://github.com/Kong/icons) library. If you wish to use a different icon library in your project, please refer to the library's documentation for instructions.
```

Reply:
I don't think `@kong/icons` is fully replaceable in Kongponents. What I was trying to say here is that if they wish to use icons exported by `@kong/icons` in their project (as of, outside of Kongponents) they can refer to `@kong/icons` documentation for installation and usage instructions.

Reply:
Applied